.gitignore

```diff
@@ -1,5 +1,5 @@
-# aragon-cli
-*/**/artifact.json
+# buidler
+ **/**/dist
```

Comment:
Does buidler product a `dist/` folder? In any case, it should be caught by the `*/*/dist` rule below?